include/inlines.h

```diff
@@ -161,14 +159,14 @@ static inline void inline_02049FA8 (UnkStruct_02049FA8 * param0, int param1, int
     param0->unk_10 = param5;
 }
 
-static inline u16 * inline_0204FCAC (UnkStruct_0203E724 * param0)
+static inline u16 * ScriptContext_GetVarPointer (ScriptContext * ctx)
```

Comment:
Should these live instead in `script.h`?

Reply:
That would require `#include "unk_0203E880.h"` in `script.h`, which will not compile since `unk_0203E880.h` already does `#include "script.h"`